src/App.js

```diff
@@ -1,29 +1,59 @@
-import logo from './logo.svg';
+/* eslint linebreak-style: 0 */
+import React, { Component } from 'react';
 import './App.css';
+import { Button } from './components/Button';
+import { Input } from './components/Input';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      input: '',
+    };
+  }
+
+  render() {
+    return (
+      <div className="app">
+        <div className="calc-wrapper">
+          {/* eslint-disable-next-line react/destructuring-assignment */}
+          <Input input={this.state.input} />
+          <div className="row">
+            <Button>AC</Button>
+            <Button>+/-</Button>
+            <Button>%</Button>
+            <Button>÷</Button>
+          </div>
+          <div className="row">
+            <Button>7</Button>
+            <Button>8</Button>
+            <Button>9</Button>
+            <Button>X</Button>
+          </div>
+          <div className="row">
+            <Button>4</Button>
```

Comment:
-[ ] Hi you really did a great job and the UI is looking good but as part of the requirements, you were supposed to create a file called `Calculator.js` and create the calculator from there.

Reply:
- Created Calculator.js file.
- Moved Calculator.js in components folder.
- Refactored code from App.js to Calculator.js file.

Thank you for reviewing my code!